framework/src/utils/JsonOutputUtils.C

```diff
@@ -0,0 +1,42 @@
+//* This file is part of the MOOSE framework
+//* https://mooseframework.inl.gov
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+// MOOSE includes
+#include "JsonOutputUtils.h"
+
+namespace JsonOutputUtils
+{
+
+nlohmann::json
+anyToJson(const std::any & data)
```

Comment:
We should override `to_json` for this value instead of putting it in a new namespace. Look at JsonIO.C and JsonIO.h